docs/using/dbaas.md

```diff
@@ -23,6 +23,10 @@ From the left menu, select <i class="uil uil-database"></i> *DBaaS*.
 
 ### Add a Kubernetes cluster
 
+!!! caution alert alert-warning "Caution"
+    Ensure that you set the <i class="uil uil-cog"></i> *Configuration* → <i class="uil uil-setting"></i> *Settings* → *Advanced Settings* before creating a Kubernetes cluster. Otherwise, PMM would not monitor this cluster.
```

Comment:
```suggestion
    Ensure that you set the <i class="uil uil-cog"></i> *Configuration* → <i class="uil uil-setting"></i> *Settings* → *Advanced Settings* before creating a Kubernetes cluster. Otherwise, PMM would not monitor database clusters created into this Kubernetes cluster.
```

Reply:
If user doesn't set public address before cluster register there won't be k8s metrics (see this [comment from Nurlan](https://jira.percona.com/browse/PMM-7143?focusedCommentId=290667&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-290667))
And also all DB clusters created from that k8s cluster won't be monitored. 
So I would leave it at "Otherwise, PMM would not monitor this cluster and related DB clusters properly." or something similar.

Reply:
properly means it will still monitor. @beatahandzelova I think it should be **Otherwise, PMM would not monitor this cluster and related database clusters."

Please reaffirm...